src/app/util/process-global-message.cpp

```diff
@@ -295,6 +295,10 @@ bool emAfProcessGlobalCommand(EmberAfClusterCommand * cmd)
         // Reset message back to start
         msgIndex          = cmd->payloadStartIndex;
         appResponseLength = (cmd->mfgSpecific ? 4 : 2);
+#if defined(__clang__)
+        // clang does not auto-detect comment marker, but accepts C++17 annotation.
+        [[fallthrough]];
```

Comment:
Can we wrap this with a macro please

#if defined(__clang__)
#define FALLTHROUGH [[clang::fallthrough]]
#else
#define FALLTHROUGH
#endif

Reply:
Done.
Not really sure about layering, since the best suitable place seems to be `support/CodeUtils.h` (have more macros there), however this will be the first chip header included from src/app. I hope I do not couple things more than they should :/

Reply:
Yes, I think that's a good place.